src/physics/rapier/shapes.rs

```diff
@@ -134,7 +134,7 @@ pub(crate) fn insert_colliders_from_shapes(
         } * Transform::from_translation(Vec3::new(pos.x, pos.y, 0.));
 
         commands
-            .spawn((collider, TransformBundle::from_transform(transform)))
+            .spawn((RigidBody::KinematicPositionBased, collider, TransformBundle::from_transform(transform)))
```

Comment:
I feel like the default should be `Fixed` ?

Reply:
Although the example might require kinematic, so you'd need this to be configurable in order to make the example work...

Reply:
Ironically my game works better without any rigid body here :thinking:. I'm still confused by that. I get the feeling that rotating the entire map like I was doing is not well supported. The movements at the edges of the map are probably too large for the physics to work smoothly